devops/poolservice/poolservice.profile

```diff
@@ -0,0 +1,2 @@
+export ConnectionString=""
+export POOL_HOME="/home/carbonuser"
```

Comment:
the user running the pool in the Beta environment is "carbonuser" but in INT and PROD it's
ncrypto.  could we use the $HOME variable here?